heksher/api/v1/rules.py

```diff
@@ -264,4 +264,78 @@ async def get_rule(rule_id: int, app: HeksherApp = application):
                          feature_values=rule_spec.feature_values, metadata=rule_spec.metadata)
 
 
+class InputRuleMetadata(ORJSONModel):
+    metadata: Dict[str, Any] = Field(default_factory=dict, description="user-defined metadata of the rule")
```

Comment:
I think we aught to put some restrictions on metadata keys (especially if we want them to serve as paths). Add to old API too?